File: src/main/java/itmo/is/project/mapper/module/storage/StorageModuleBlueprintMapper.java
package itmo.is.project.mapper.module.storage;

import itmo.is.project.dto.module.storage.StorageModuleBlueprintDto;
import itmo.is.project.mapper.EntityMapper;
import itmo.is.project.model.module.storage.StorageModuleBlueprint;
import org.mapstruct.Mapper;

@Mapper
public interface StorageModuleBlueprintMapper extends EntityMapper<StorageModuleBlueprintDto, StorageModuleBlueprint> {
}

File: src/main/kotlin/itmo/isproject/storage/mapper/StorageModuleBlueprintMapper.kt
package itmo.isproject.storage.mapper

import itmo.isproject.storage.dto.StorageModuleBlueprintDto
import itmo.isproject.shared.common.mapper.EntityMapper
import itmo.isproject.storage.model.StorageModuleBlueprint
import org.mapstruct.Mapper

@Mapper
interface StorageModuleBlueprintMapper : EntityMapper<StorageModuleBlueprintDto, StorageModuleBlueprint>
